spec/unit/action/puppetfile/install_spec.rb

```diff
@@ -19,9 +19,7 @@ def installer(opts = {}, argv = [], settings = {})
 
   describe "installing modules" do
     let(:modules) do
-      (1..4).map do |idx|
-        R10K::Module::Base.new("author/modname#{idx}", "/some/nonexistent/path/modname#{idx}", nil)
-      end
+      Array.new(4, R10K::Module::Base.new('author/modname', "/some/nonexistent/path/modname", nil))
```

Comment:
looks like we might want to keep this? https://github.com/puppetlabs/r10k/pull/1044/commits/0e198cac586cf79af5ad1ce0b1305a3eb02846e1

Reply:
Updated